src/view/ViewCommandHandlers.js

```diff
@@ -139,6 +145,18 @@ define(function (require, exports, module) {
      */
     function _removeDynamicFontSize() {
         _removeDynamicProperty(DYNAMIC_FONT_STYLE_ID);
+        _removeDynamicProperty(DYNAMIC_CODEHINT_FONT_STYLE_ID);
+    }
+
+    /**
+     * @private
+     * Adds a new embeded style top sync code-hint font size with codeview font size
+     */
+    function _addDynamicFontSizeForCodeHints(fontSize) {
+        var styleStr = "";
+        styleStr = styleStr + StringUtils.format("{0}: {1} {2};", "font-size", fontSize, " !important");
+        styleStr = styleStr + StringUtils.format("{0}: {1} {2};", "line-height", (parseInt(fontSize, 10) + 2) + fontSize.replace(parseInt(fontSize, 10), ""), " !important");
+        _addDynamicProperty(DYNAMIC_CODEHINT_FONT_STYLE_ID, "font-size", fontSize, true, ".codehint-menu .dropdown-menu li a", styleStr);
```

Comment:
I think `function _addDynamicProperty(propertyID, options)` would make more sense (where options would be an object of `{ name, value, important, cssRule, cssText }` etc.)